packages/astro/src/index.ts

```diff
@@ -1,46 +1,49 @@
-import type { AstroIntegration, ViteUserConfig } from 'astro';
-import { fileURLToPath } from 'node:url';
-import path from 'node:path';
-import fs from 'node:fs/promises';
+import type { AstroIntegration } from 'astro';
+import { mkdirSync, writeFileSync } from 'node:fs';
 
 export default function keystatic(): AstroIntegration {
   return {
     name: 'keystatic',
     hooks: {
-      'astro:config:setup': async ({ injectRoute, updateConfig, config }) => {
-        if (config.output !== 'hybrid') {
+      'astro:config:setup': ({ injectRoute, updateConfig, config }) => {
+        if (config.output === 'static') {
```

Comment:
As long as `prerender` can be `false`, it should be usable by the users

Reply:
Yup, I think that makes sense to allow both `hybrid` and` server` — as long as the Keystatic routes themselves are not static 🤙